src/App.test.tsx

```diff
@@ -5,6 +5,8 @@ import App from './App';
 import styles from './App.module.scss';
 
 it('renders without crashing', () => {
+  fetchMock.mockResponse(JSON.stringify({}));
```

Comment:
Yes, because I added an api call directly in a lifecycle method, so it is invoked in this test case. That's a demo, but better have a green test suite :p

Reply:
If I comment out this line the test still passes.

Reply:
Yep, but it should also output a warning about a promise not resolved (or something like this)

Reply:
removed